lib/locales/br/frontend/api-messages.json

```diff
@@ -531,8 +531,5 @@
   "E97": "Falha ao criar categoria do desafio",
   "E98": "Falha na atualização do desafio",
   "E99": "Falha na marcação da atualização do desafio",
-  "E610": "Falha ao seguir {{entity}}",
-  "E611": "Não permitido seguir este(a) {{entity}}",
-  "E612": "Falha ao deixar de seguir este(a) {{entity}}",
```

Comment:
Maybe you should ignore the api-messages, cause these codes are emitted by the API

Reply:
Thnak you. I also included the `countries`